.circleci/config.yml

```diff
@@ -0,0 +1,46 @@
+# We are supposed to use architect-orb to build&push container images.
+# However, architect-orb has strict rules and tests. We don't have full control over this repository.
+# That is why we introduced this custom configuration in this repository.
+# Keep this file as it is while merging changes from upstream repository.
+
+version: 2.1
+
+jobs:
+  build:
+    machine:
+      image: ubuntu-2204:2022.04.2
+    steps:
+      - checkout
+      - run:
+          name : Build container
+          command: |
+            export IMG=capvcd:latest
+            make docker-build
+           
+      - run:
+          name: Push to quay.io
+          command: |
+            echo "$QUAY_PASSWORD" | docker login quay.io --username $QUAY_USERNAME --password-stdin
+            
+            IMG_TAG=$(git rev-parse --short HEAD)
+            docker tag capvcd:latest quay.io/giantswarm/cluster-api-provider-cloud-director-vcd:$IMG_TAG
+            docker push quay.io/giantswarm/cluster-api-provider-cloud-director-vcd:$IMG_TAG
+
+            echo "Pushed to quay.io/giantswarm/cluster-api-provider-cloud-director-vcd:$IMG_TAG"
+
+      - run:
+          name: Push to docker.io
+          command: |
+            echo "$DOCKER_PASSWORD" | docker login --username $DOCKER_USERNAME --password-stdin
+
+            IMG_TAG=$(git rev-parse --short HEAD)
+            docker tag capvcd:latest docker.io/giantswarm/cluster-api-provider-cloud-director-vcd:$IMG_TAG
+            docker push docker.io/giantswarm/cluster-api-provider-cloud-director-vcd:$IMG_TAG
+
+            echo "Pushed to docker.io/giantswarm/cluster-api-provider-cloud-director-vcd:$IMG_TAG"
+workflows:
+  build-workflow:
+    jobs:
+      - build:
+          context:
+            - architect
```

Comment:
I understand all QUAY* and DOCKER* variables come from the context?

Reply:
Yes.